ramls/request.json

```diff
@@ -74,6 +74,12 @@
       "readOnly": true,
       "type": "object",
       "properties": {
+        "status": {
```

Comment:
Is this a new property? If so, why is it being introduced?

Reply:
Same as above, it was missed in the schema definition, see [RequestRepresentation.java#L87](https://github.com/folio-org/mod-circulation/blob/master/src/main/java/org/folio/circulation/domain/RequestRepresentation.java#L87)

Reply:
Similarly, was it introduced intentionally and the bug was in the interface definition or accidentally because of shared representation classes and the bug is in the implementation?

Reply:
I'm not sure, the property was added almost 2 years ago. It seems that it is an interface issue.

Reply:
Ok, I'll close resolve